src/bin.ts

```diff
@@ -91,6 +103,56 @@ const createSafeS3Key = (key: string): string => {
     return key;
 };
 
+const uploadQueue = queue((task: UploadTask, callback: (key?: string, err?: any) => void) => {
```

Comment:
I think uploadQueue should be created/scoped within the deploy function, since it is populated within that function and is drained within that function.